inc/customizer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * Adding logo and theme options to customizer
```

Comment:
I'm trying to figure out what our concept for document titles is. Honestly, I'm not 100% sure. But it looks like we're mostly using the imperative.
```suggestion
 * Add theme options to customizer
```